doc/ip/system_info.md

```diff
@@ -0,0 +1,11 @@
+# System information
+
+This is a special IP block that is automatically populated with useful information about the system.
+These values are useful to know what the parameters were when a bitstream was generate.
+
+| Offset | Register |
+|--------|----------|
+| 0x0000 | Git commit hash from which this bistream was generated.
+| 0x0004 | Git dirty status (1 if dirty or 0 if clean).
+| 0x0008 | System clock frequency in Hz.
```

Comment:
Oh no, sonata is not limited to <4.3 GHz in perpetuity!

Reply:
I was thinking about this, but didn't find this a good enough reason to switch to kHz ;)